jobbergateapi2/apps/job_scripts/routers.py

```diff
@@ -90,27 +96,15 @@ async def job_script_create(
         "templates/" + default_template,
     ]
 
-    application_location = (
-        f"{settings.S3_BASE_PATH}/TEST/applications/{application.id}/jobbergate.tar.gz"
-        # f"{S3_BASE_PATH}/{application.owner_id}/applications/{application.id}/jobbergate.tar.gz"
-    )
-    try:
-        s3_application_obj = s3_client.get_object(Bucket=S3_BUCKET, Key=application_location)
-    except BotoCoreError:
-        raise HTTPException(
-            status_code=404,
-            detail=f"Application with id={application_id} not found for user={current_user.id} in S3",
-        )
-    s3_application_tar = tarfile.open(fileobj=BytesIO(s3_application_obj["Body"].read()))
     template_files = {}
     for member in s3_application_tar.getmembers():
         if member.name in default_template:
             contentfobj = s3_application_tar.extractfile(member)
             template_files["application.sh"] = contentfobj.read().decode("utf-8")
         if member.name in supporting_files:
-            match = [x for x in support_files_ouput if member.name in x]
+            match = [x for x in support_files_output if member.name in x]
             contentfobj = s3_application_tar.extractfile(member)
-            filename = support_files_ouput[match[0]][0]
+            filename = support_files_output[match[0]][0]
             template_files[filename] = contentfobj.read().decode("utf-8")
```

Comment:
This is not ideal yet, and will likely be removed in the future, but the focus now is to keep it working like the original jobbergate-api first